### DISCUSSION TITLE
special tiff format?

### DISCUSSION THREAD

**Discussion by ephr4321** [emgucv/emgutf#86] [General]
Hi,
I have this tiff file, which almost no viewer opens. BUT irfan viewer manages, and dropbox preview also manages.
EMGU fails to read it.

[badTiff.zip](https://github.com/emgucv/emgutf/files/11241692/badTiff.zip)

Any ideas?
Thanks!

**Comment by emgucv** [comment]
It cannot be read by Open CV python:

```
>>> import cv2
>>> im = cv2.imread("badTiff.tif")
>>> print(type(im))
<class 'NoneType'>
```

You will need to fix the tiff file such that Open CV can read it. If Open CV python cannot read it, Emgu CV won't be able to read it.
